static/script.js

```diff
@@ -5,7 +5,33 @@ $(function () {
     dateTime.local()
     $element.text(dateTime.format('YYYY-MM-DD HH:mm:ss'))
   })
+
   $('.no-collapsable').on('click', function (event) {
     event.stopImmediatePropagation()
   })
 })
+
+function RefreshMe () { // eslint-disable-line no-unused-vars
```

Comment:
Please change function name to camelCase (i. e. `refreshMe`). Also maybe change the name to `enableAutoRefresh`.